blockchain/blocks/src/tipset.rs

```diff
@@ -133,6 +134,27 @@ impl Tipset {
     pub fn weight(&self) -> &BigInt {
         self.min_ticket_block().weight()
     }
+    /// Returns true if self wins according to the filecoin tie-break rule (FIP-0023)
+    pub fn break_weight_tie(&self, other: &Tipset) -> bool {
+        // blocks are already sorted by ticket
+        let broken = self
+            .blocks()
+            .iter()
+            .zip(other.blocks().iter())
+            .any(|(a, b)| {
+                const MSG: &str =
+                    "The function block_sanity_checks should have been called at this point.";
+                let ticket = a.ticket().as_ref().expect(MSG);
+                let other_ticket = b.ticket().as_ref().expect(MSG);
+                ticket.vrfproof < other_ticket.vrfproof
+            });
+        if broken {
+            info!("weight tie broken in favour of {:?}", self.key());
```

Comment:
should it be `info` or rather `debug`? Not sure how often should we see it and how relevant it is for anyone running the node.

Reply:
Or maybe `warn` if this is something that is highly unexpected.

Reply:
I would prefer to let it as `info` first so that it's really visible to all of us, then turn it to `debug` or `warn` yes.
It's probably an event that is very likely to happen.